README.md

```diff
@@ -90,16 +91,19 @@ autobib source main.tex --out main.bib
 ```
 will search through the document for valid citation keys and output the bibliography into the file `main.bib`.
 
-### Editing records
+### Modifying records
 
 On the first run, Autobib retrieves the data directly from a remote provider.
 The data is stored locally in a [SQLite](https://www.sqlite.org/) database, which defaults to `~/.local/share/autobib/records.db`, so that subsequent runs are substantially faster.
-You can view and update the internally stored record with
+You can view and modify the internally stored record with
 ```bash
 autobib edit zbl:1337.28015
 ```
 If the record does not yet exist in your local record database, it will be retrieved before editing.
 
+You can also re-retrieve a record from the remote provider using the `autobib update` command, or remove one from the database using the `autobib delete` command.
+Run `autobib help update` and `autobib help delete` for more details.
```

Comment:
'for more detail'?

Reply:
I think "for more details" is fine. https://dictionary.cambridge.org/dictionary/english/detail

Reply:
I guess I just prefer detail to details! But I agree both are grammatically OK.

Reply:
I'll go with the majority opinion😬 https://books.google.com/ngrams/graph?content=for+more+detail%2Cfor+more+details&year_start=1800&year_end=2022&corpus=en&smoothing=3